spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/aws/AWSBedrockAutoConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package org.springframework.ai.autoconfigure.aws;
+
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.experimental.ai.client.AWSBedrockClient;
+
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.services.bedrockruntime.BedrockRuntimeClient;
+
+@AutoConfiguration
```

Comment:
I would use the https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/bedrock/BedrockClient.html or the BedrockAsyncClient here. I think just moving into the latest version of the SDK gives you a lot of new classes to use.

Reply:
So just to be sure : the `BedRockClient` provides methods for e.g. deploying models. 
What I currently implements is a way of interacting with the model 
As much as I undersstand `BedRockRuntimeClient` should be correct one to do that right? 
💯  to updating the SDK version

Reply:
You are correct, https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/bedrockruntime/BedrockRuntimeClient.html

thanks